icecream/icecream.py

```diff
@@ -165,13 +174,13 @@ def detect_terminal_width(prefix, default=DEFAULT_LINE_WRAP_WIDTH):
     """ Returns the number of columns that this terminal can handle. """
     width = default
     try:
-        if hasattr(shutil, "get_terminal_size"):
-            width = shutil.get_terminal_size().columns
-        else:  # Python 2.x doesn't support get_terminal_size
-            from backports.shutil_get_terminal_size import get_terminal_size
-            width = get_terminal_size().columns
-    except OSError:  # Not in TTY
+        # We need to pass a terminal height in the tuple so we pass the default
+        # of 25 lines but it's not used for anything.
+        width = get_terminal_size((default, 25)).columns
```

Comment:
```suggestion
        width = get_terminal_size((default, 25))[0]
```

to work with the last resort `get_terminal_size` without having to involve namedtuples or something.